### ISSUE TITLE
Hume's rainbow cave: Missing level licenses

### ISSUE THREAD

**Issue by Wuzzy2** [SuperTux/addons#9]
Bug in Hume's rainbow cave in SuperTux 0.4.0:

In developer mode, SuperTux is complaining about missing licenses for a couple of levels.

**Comment by maxteufel** [comment]
Assigning all of #8, #9 and #10 to @Hume2. If they don't want to fix that, sorry. We don't provide official support for add-ons (else they would be part of the game).

**Comment by Hume2** [comment]
@Wuzzy2 Yes, you can copy it and manipulate with it as you wish, you can even sell it, burn it, print it, copy it again, edit it, put it a license, scrap it, delete it, sell it again, whatever you want, I have no profit from it and I won't blame you if you did anything with that, because firstly I'm not crazy about licenees, secoundly I'm not an idiot, thirdly I can't control you and fourthly, I'm not crazy about licenses. I'd most likely close this issue because it's really a crap, but I can't. I'll try to put here a "license" in a bugfix update of my worlds, please remind it me if I don't make it in the next bilion years. Even I don't like obeying the licenses, so why do you care?

*Kde není žalobce, není soudce*
Yes, translate it yourself.

**Comment by maxteufel** [comment]
Just put it under Public Domain (or CC0, Unlicense, WTFPL) then.

**Comment by Wuzzy2** [comment]
lol. I just posted this because SuperTux annoyed me. XD

**Comment by LMH0013** [comment]
I just downloaded the addon and there are still unspecified licenses.  Specifying a license is how you let others know that you are okay with any use of it.  It is just basic good communication and prevents others from worrying about what they can and cannot do with your creations.
